src/btree/bt_curnext.c

```diff
@@ -749,10 +754,10 @@ __wt_btcur_next_prefix(WT_CURSOR_BTREE *cbt, WT_ITEM *prefix, bool truncating)
     WT_SESSION_IMPL *session;
     size_t total_skipped, skipped;
     uint32_t flags;
-    bool key_out_of_bounds, newpage, restart, need_walk;
+    bool key_out_of_boundsp, newpage, restart, need_walk;
 
     cursor = &cbt->iface;
-    key_out_of_bounds = false;
+    key_out_of_boundsp = false;
```

Comment:
Ah - my earlier request wasn't clear. It turns into a `p` when it becomes a pointer. So the local bool variable here would be `key_out_of_bounds`, then it gets passed by reference i.e: turned into a pointer.

i.e: the local variable is `key_out_of_bounds`. You tried to ask me this in Slack and I gave you the wrong answer, I think. Apologies for making work.

Reply:
As mentioned in our brief offline chat, I think I asked you in a confusing manner earlier on Slack, so no worries about that at all! :) I understand what needs to be done, and have just made those changes.